### PR TITLE
Add run-status in bundle sidebar view

### PR DESCRIPTION
**Pull request by pranavjain**
Fixes part of #4085 

image

### CODE REVIEW

**Review by epicfaace — COMMENTED** [review]
thanks -- can you add a screenshot?

**Comment by pranavjain** [comment]
> thanks -- can you add a screenshot?

Uploaded in the pull request description. @epicfaace

**Review by epicfaace — APPROVED** [review]

File: frontend/src/components/worksheets/BundleDetail/BundleDetailSideBar.jsx [hunk 1]
```diff
@@ -146,6 +147,13 @@ class BundleDetailSideBar extends React.Component<{
                     )}
                 </div>
                 {/** ----------------------------------------------------------------------------------------------- */}
+                {isRunBundle && runStatus && (
+                    <div>
+                        <ConfigLabel label='Run status: ' inline={true} />
+                        <div className={classes.dataText}>{runStatus}</div>
```
Comment (epicfaace) [inline frontend/src/components/worksheets/BundleDetail/BundleDetailSideBar.jsx:153]: Will it always be red in color?

**Review by epicfaace — APPROVED** [review]

File: frontend/src/components/worksheets/BundleDetail/BundleDetailSideBar.jsx (hunk 1 above)
Comment (pranavjain) [reply frontend/src/components/worksheets/BundleDetail/BundleDetailSideBar.jsx:153]: It won't ever be red in color (same as shown in the screenshot).

**Pull request merged**